libcasr/src/stacktrace.rs

```diff
@@ -287,11 +288,226 @@ impl Filter for Stacktrace {
             self.drain(pos + 1..);
         }
 
-        // Remove trusted functions from stack trace
+        // Find repeating intervals in stacktrace
+        let mut indices = Vec::new();
+        indices.resize(self.len(), true);
+        // We need to create an element that will not be equal to others
+        let mut not_an_element = StacktraceEntry::default();
+        not_an_element.debug.file = "not_equal_to_other_names".to_string();
+        not_an_element.debug.line = 1;
```

Comment:
Can we create `not_an_element` inside `get_interval_repetitions` function?

Reply:
`get_interval_repetitions` is now written with generics for convenience, so i don't know how to create `not_an_element` for type `T`